src/rules/noRequireImportsRule.ts

```diff
@@ -20,15 +20,33 @@ import * as ts from "typescript";
 
 import * as Lint from "../index";
 
+const OPTION_IGNORE_MODULE = "ignore-module";
+
 export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "no-require-imports",
         description: "Disallows invocation of `require()`.",
         rationale: "Prefer the newer ES6-style imports over `require()`.",
         optionsDescription: "Not configurable.",
```

Comment:
I don't know how this property is used but with the new option the rule is in fact configurable, no?